.github/workflows/review_pull_request.yml

```diff
@@ -30,16 +30,16 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-gradle-
 
-      - name: Run Detekt on template
-        working-directory: ./template
+      - name: Run Detekt on template-xml
+        working-directory: ./template-xml
         run: ./gradlew detekt
 
-      - name: Run Android Lint on template
-        working-directory: ./template
+      - name: Run Android Lint on template-xml
+        working-directory: ./template-xml
         run: ./gradlew lint
 
-      - name: Run unit tests with Kover on template
-        working-directory: ./template
+      - name: Run unit tests with Kover on template-xml
+        working-directory: ./template-xml
```

Comment:
@Wadeewee Wait, This workflow is in the big `android-template`, isn't it? 
If we update on this level, I am concerned that this workflow will not work with the `template-compose`. We probably need to separate workflow files for `template-xml` and `template-compose`. What do you think? cc @Tuubz 

<img width="379" alt="Screen Shot 2565-11-17 at 10 14 47 AM" src="https://user-images.githubusercontent.com/12026942/202346070-8eef32e9-0ef7-42ec-9c90-e04945e279d7.png">

Reply:
The step for `template-compose` will be run later, once @chornerman integrates #333

Reply:
@Tuubz Hmmm 😅  Then finally the workflow will have only for `template-compose`, right? 😂 Imagine when someone updates the `template-xml`, the workflow will not run for it. 🤔  Is that your intention?

Reply:
@lydiasama No, the workflow will have both `template-compose` & `template-xml` 

A new step will be added for `template-compose` in #333

Did I miss something? 🙇

Reply:
@Tuubz Then I think it's time to rename this file to `xml_review_pull_request` and for #333 will not be update but create a new workflow `compose_review_pull_request`. Same to the rest workflows. What do you think?

Reply:
Hmmm, any reason we can't keep it in one file? We used to do the same for `RxJava` & `Coroutine` template  🙏

Reply:
@Tuubz Oh, I might be wrong. If we can create it in one file, it would be best. 👍🏻